js/textbox/shared/Textbox.svelte

```diff
@@ -246,7 +251,12 @@
 			/>
 		</IconButtonWrapper>
 	{/if}
-	<BlockTitle {show_label} {info}>{label}</BlockTitle>
+	<BlockTitle show_label={validation_error ? true : show_label} {info}
```

Comment:
This is because ChatInterface doesn't show a label but we should show the validation_error if it exists